test/base.py

```diff
@@ -51,8 +51,12 @@ def setup_class(cls):
     def config_template(self, override=None):
         # NOTE: we set pg_receivexlog_path and pg_basebackup_path per site and globally mostly to verify that
         # it works, the config keys are deprecated and will be removed in a future release at which point we'll
-        # switch to using pg_bin_directory config.
-        bindir, ver = find_pg_binary("")
+        # switch to using pg_bindirectory config.
```

Comment:
This line looks like it should not be changed

Reply:
Done